app/src/main/java/pm/gnosis/heimdall/ui/settings/general/changepassword/ChangePasswordViewModel.kt

```diff
@@ -12,10 +14,23 @@ import javax.inject.Inject
 class ChangePasswordViewModel @Inject constructor(
     private val encryptionManager: EncryptionManager
 ) : ChangePasswordContract() {
-    override fun validatePassword(password: String): Single<Result<Collection<PasswordValidationCondition>>> =
+
+    private val confirmSubject = PublishSubject.create<Unit>()
+    private val viewState = PublishSubject.create<ViewState>()
+    private var _viewState = ViewState(State.ENTER_OLD_PASSWORD, true)
```

Comment:
We never read `_viewState`, do we need it?

Reply:
no, removing